test/unit/puppetlabs/puppetserver/certificate_authority_test.clj

```diff
@@ -448,6 +466,20 @@
       (revoke-existing-cert! settings "localhost")
       (is (true? (revoked? cert))))))
 
+(deftest revoke-with-bundled-ca-certs
+  (testing "The CA certificate file can be a bundle when revoking a certificate"
+    (let [settings (testutils/ca-sandbox! cadir)
+          cert     (-> (:signeddir settings)
```

Comment:
I honestly don't care much about formatting but it was my understanding that we only put one space between keys and values in maps and names and values in assignment (and that the formatting in the CA was especially old).